components/resource_metering/src/config.rs

```diff
@@ -40,7 +40,7 @@ impl Default for Config {
         Config {
             receiver_address: "".to_string(),
             report_receiver_interval: ReadableDuration::minutes(1),
-            max_resource_groups: 2000,
+            max_resource_groups: 200,
```

Comment:
Please change to 100~

Reply:
Has been modified before, try to refresh :)